typescript/web/src/components/graphics/no-workspace.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react";
+import { SVGProps, Ref, forwardRef, memo } from "react";
+
+const SvgNoWorkspace = (
+  props: SVGProps<SVGSVGElement>,
+  ref: Ref<SVGSVGElement>
+) => (
+  <svg
```

Comment:
I don't understand why do we have to do store the SVG twice in the repository:
* 1 in the static directory (_1st file in the diff of this PR_). Is it really useful? Or is it to have a backup of the source SVG?
* 1 in a React component (_which makes it unusable outside of our app_)
I know other images in this directories have been declared the same way but I think that in the long-term we should find a way to avoir such duplications.